src/main/java/com/candyshop/islodycze/model/ProductOrder.java

```diff
@@ -31,7 +29,8 @@ public class ProductOrder {
     private Product productFk;
 
     @JsonIgnore
-    @ManyToMany(mappedBy = "productOrderFk")
+    @ManyToOne
```

Comment:
Why did you resign from many to many relation?

Reply:
We don't want to have many to many realtion with ProductOrder table, but with Order. So i create ManyToOne to ProductOrder.